hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/NativeSparkDataSet.java

```diff
@@ -838,6 +860,27 @@ private String planIfLast(AbstractSpliceFunction f, boolean isLast) {
         }
     }
 
+    /**
+     * Take a Splice DataSet in the consumer's context, with a single source,
```

Comment:
s/DataSet/Dataset

Reply:
Splice calls it DataSet, not Dataset.

`/**
 * Stream of data acting on an iterable set of values.
 */
public interface DataSet<V> extends //Iterable<V>,
        Serializable {`